daemon/algod/api/server/v2/handlers.go

```diff
@@ -957,14 +967,43 @@ func (v2 *Handlers) SimulateTransaction(ctx echo.Context, params model.SimulateT
 	}
 	if stat.Catchpoint != "" {
 		// node is currently catching up to the requested catchpoint.
-		return serviceUnavailable(ctx, fmt.Errorf("SimulateTransaction failed as the node was catchpoint catchuping"), errOperationNotAvailableDuringCatchup, v2.Log)
+		return serviceUnavailable(ctx, fmt.Errorf("ExtendedSimulateTransaction failed as the node was catchpoint catchuping"), errOperationNotAvailableDuringCatchup, v2.Log)
```

Comment:
What is extending exactly.
ParametricSimulate?
AdvancedSimulate?
SimulateWithOptions?
SimulatePlus?

Reply:
@algoanne

Reply:
This was meant to change back to "SimulateTransaction." Thanks for catching